tests/characterization_data/filters_expected/sample_nsxv.nsx

```diff
@@ -1,54 +1,612 @@
-<!--
- $Id:$
- $Date:$
- $Revision:$
--->
+<?xml version="1.0" ?>
+<!-- $Id:$ $Date:$ $Revision:$-->
```

Comment:
The new lines were lost for these keywords.  We need to keep that newline structure.